.vscode/settings.json

```diff
@@ -3,5 +3,11 @@
 		"*.css": "tailwindcss"
 	},
 	"typescript.tsdk": "./core/node_modules/typescript/lib",
-	"javascript.preferences.autoImportSpecifierExcludeRegexes": ["@prisma\\/client"]
+	"javascript.preferences.autoImportSpecifierExcludeRegexes": ["@prisma\\/client"],
+	// these variables are necessary in order to correctly run seeds
+	"playwright.env": {
+		"NODE_OPTIONS": "--import=\"#register-loader\"",
```

Comment:
this weird `#` syntax is needed because the extension run playwright in two separate directiories, `/` and `/core`. So i couldn't just have it point to `./prisma/seed/stub/register-loader.js`, bc that would not work for the one running in `/`, ughhh.

so instead, i set an `imports.#register-loader` in both `/package.json` and `core/package.json`, pointing to the same file slightly differently. this seems to work.